test/box/error.result

```diff
@@ -449,6 +449,7 @@ t;
  |   228: box.error.SYNC_QUEUE_FOREIGN
  |   229: box.error.UNABLE_TO_PROCESS_IN_STREAM
  |   230: box.error.UNABLE_TO_PROCESS_OUT_OF_STREAM
+ |   231: box.error.TRANSACTION_TIMEOUT
```

Comment:
I don't see any tests. Please don't add untested code unless absolutely necessary. Squash patches if you have to.

Also, the doc bot request doesn't make any sense - there's nothing to document, because there's no user API for now. Looks like this patch needs to be squashed with the next one.

Reply:
Ok, fixed  squash patches